src/main/kotlin/anttracker/Release.kt

```diff
@@ -23,33 +26,115 @@ value class ReleaseId(
             "Release id length must be between 1 and 8 characters"
         }
     }
+
+    override fun toString(): String = id
 }
 
-class Release(
-    val releaseName: ReleaseId,
-    val product: Product,
-    var releaseDate: LocalDate,
-)
+val promptEnterRel = "\nPlease enter new release name. ` to abort:"
+
+val promptSelectRel = "\nPlease select release. ` to abort:"
+
+val promptSelectAffRel = "\nPlease select affected release. ` to abort:"
 
 // -------------------------------------------------------------------------------
 // Prints a sub-menu of selecting a product to create a release for.
 // Use by including in main-menu loop; should be triggered upon user selecting
 // New Release from the main menu. Thus, upon function return, system returns to main menu.
 // ---
 fun menu() {
-    TODO()
+    println("== NEW RELEASE ==")
+    // val selectedProduct = selectProduct()    <- TODO: uncomment and replace when implemented in product module
```

Comment:
I'd recommend leaving this uncommented for now. If this function isn't written in the Product.kt module, feel free to add it and put TODO() in the function body

Reply:
Also, you should do a null check here, if the user indicates to exit early in selectProduct. If selectProduct returns null, menu should return null as well, returning to the main menu.

Reply:
Agreed with all.